tests/Responses/Expected/order_item/create_response.json

```diff
@@ -0,0 +1,27 @@
+{
+    "id": @integer@,
+    "quantity": 1,
+    "unit_price": 2000,
+    "total": 2000,
+    "immutable": @boolean@,
```

Comment:
I bet, this value can be determined from the beginning, so I would vote to create different create_response with different values declared, rather then creating on file with lot of matcher. By default it would be better if we would keep matcher for things, that can be changed really easily, such as date or id